docs/emu_mps/notebooks/dmrg_usecase.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c9ca6bc2",
+   "metadata": {},
+   "source": [
+    "# DMRG tutorial: Getting started\n",
+    "\n",
+    "The aim of this tutorial is to show how the **Density Matrix Renormalization Group (DMRG)** algorithm can be used to simulate Pulser sequences.\n",
```

Comment:
Plz, say that DMRG is used to replicate time evolution as if total duration is infinite. And why we expect it to work